### PR TITLE
Update Readme with initializing information for Edge extensions

### PR DESCRIPTION
**Pull request by cacheung**
Update Readme with initializing information for  Edge extensions

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/edge/README.md [hunk 1]
```diff
@@ -31,6 +31,65 @@ Install the Adobe Experience Platform Edge Network extension in your mobile prop
...
+@implementation AppDelegate
+
+// AppDelegate.m
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [AEPMobileCore registerExtensions:@[AEPMobileEdgeIdentity.class, 
+                                        AEPMobileEdge.class] completion:^{
+     ...	
+
+   }];
+   [AEPMobileCore configureWithAppId:@"yourAppID"];
```
Comment (emdobrin) [inline packages/edge/README.md:53]: Let's move this inside the completion block for registerExtensions.
Also, do we need to add the iOS setWrapperType example too?

File: packages/edge/README.md [hunk 2]
```diff
@@ -31,6 +31,65 @@ Install the Adobe Experience Platform Edge Network extension in your mobile prop
...
+
+    try {
+      Edge.registerExtension();
+      com.adobe.marketing.mobile.edge.identity.Identity.registerExtension();
+      MobileCore.start(new AdobeCallback() {
+        @Override
+        public void call(Object o) {
+        
+        }});
+    } catch (InvalidInitException e) {
```
Comment (emdobrin) [inline packages/edge/README.md:86]: can we remove the try catch? I believe it is not required for the Edge extensions

**Review by emdobrin — COMMENTED** [review]
Changes look good, added couple small comments that apply for all extensions

File: packages/edge/README.md (hunk 1 above)
Comment (cacheung) [reply packages/edge/README.md]: Add  setWrapperType to iOS.
Do we need lifecycle in the initializing example?

File: packages/edge/README.md [hunk 3]
```diff
@@ -31,6 +31,65 @@ Install the Adobe Experience Platform Edge Network extension in your mobile prop
 Then follow the same document for registering the Edge extension with the Mobile Core.
 Note that initializing the SDK should be done in native code, additional documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).
 
+
+Initializing Example:
```
Comment (emdobrin) [inline packages/edge/README.md:35]: Let's rename Initializing Example: -> **Initialization example** - same for all files

File: packages/edge/README.md [hunk 4]
```diff
@@ -31,6 +31,65 @@ Install the Adobe Experience Platform Edge Network extension in your mobile prop
...
+...
+@implementation AppDelegate
+
+// AppDelegate.m
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [AEPMobileCore setLogLevel: AEPLogLevelDebug];
+    [AEPMobileCore setWrapperType: AEPWrapperTypeReactNative];
+    [AEPMobileCore registerExtensions:@[AEPMobileEdgeIdentity.class, 
+                                        AEPMobileEdge.class] completion:^{
+    [AEPMobileCore configureWithAppId:@"yourAppID"];
```
Comment (emdobrin) [inline packages/edge/README.md:52]: Formatting: can you add a tab for this line? - Same for all ios examples

**Review by emdobrin — COMMENTED** [review]
Small formatting suggestions, looks great otherwise

File: packages/edge/README.md (hunk 1 above)
Comment (emdobrin) [reply packages/edge/README.md]: No, that is covered in the core examples.

**Review by emdobrin — APPROVED** [review]

**Pull request merged**